widget/toolbar.go

```diff
@@ -112,6 +120,15 @@ func NewToolbar(items ...ToolbarItem) *Toolbar {
 	return t
 }
 
+func (t *Toolbar) Disable() {
```

Comment:
Please add a comment to this public method. It also needs a `Since: ` line at the bottom.

Reply:
Hi @Jacalz, thanks for the review. I'll modify the code to include the above mentioned changes

Reply:
Thanks. FYI, you don't need to tag any of us unless it is unclear who you are directing your message too. Not doing so is actually preferred because we don't have to notified directly but can look at things when we have time. We have a project dashboard where we see al new PRs and we get notifications for discussions we are taking part in.

Reply:
Sorry but you added the wrong version in the since line. 2.4 has already been released. The upcoming feature  release is 2.5.